apps/backend/src/modules/groups/group.entity.ts

```diff
@@ -52,34 +55,49 @@ class GroupEntity implements Entity {
 		users,
 	}: {
 		name: string;
-		permissions: Pick<PermissionModel, "id">[];
+		permissions: Array<{ id: number }>;
 		users: Pick<UserModel, "id">[];
 	}): GroupEntity {
 		return new GroupEntity({
+			createdAt: null,
 			id: null,
 			name,
 			permissions,
 			users,
 		});
 	}
 
+	public toGetAllItemObject(): GroupGetAllItemResponseDto {
```

Comment:
We need to modify toObject instead, in our case we will need to have the same entity fields for get all as for create, because after creation we are going to insert returned item to the store
```suggestion
```